platformio/package/download.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, url, dest_dir=None):
         self._request = requests.get(
             url,
             stream=True,
+            verify=False,
```

Comment:
This weakens security for *everyone* not using a proxy, opening them up to man-in-the-middle attacks, supply chain poisoning etc. Can we only ocnditionally do this if another option / environment variable is enabled, such as "Disable strict SSL verifcation"? Same for the line above.

Reply:
Of course, like this it's a big security concern ;)

We could base this on the environment variable [NODE_TLS_REJECT_UNAUTHORIZED](https://nodejs.org/api/cli.html#node_tls_reject_unauthorizedvalue) ?

Reply:
Yes, we will provide environment variable to disable TLS Certificates verification